### PR TITLE
Update sizing cluster article to ping external article

### PR DESCRIPTION
**Pull request by pipo02mix**
After finding this good article on learnk8s we want to reference it in our docs

### What does this PR do?

(Please set a descriptive PR title. Use this space for additional explanations.)

Reviews the multi-tenancy cluster sizing article 

### What does it look like?

(If this is more than a textual change, a screenshot can help in some cases to speed up the review process.)

### Any background context you can provide?

(Please link public issues or summarize if not public.)

### Have you maintained the front matter?

(Please bump the last_review_date in case this qualifies as a review for an entire page. Provide user_questions which should be answered in the page. Provide a meaningful description.)

### CODE REVIEW

File: src/content/getting-started/sizing-multi-tenancy/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ user_questions:
   - How should I separate concerns using several clusters?
   - How many worker nodes should my clusters have?
   - How many control plane nodes should I run?
-last_review_date: 2022-12-07
+last_review_date: 2023-28-08
```
Comment (AndiDog) [inline src/content/getting-started/sizing-multi-tenancy/index.md:21]:
```suggestion
last_review_date: 2023-08-28
```

File: src/content/getting-started/sizing-multi-tenancy/index.md [hunk 2]
```diff
@@ -30,17 +30,16 @@ Keep in mind that these recommendations are just basic rules-of-thumb that you s
...
+Otherwise, you should always account for at least 1 node buffer, so in case one of your nodes goes down or gets rolled due to updates your workloads still have room to be scheduled.
 
-Furthermore, once you want to actually use the cluster productively, we would recommend to have at least 4 worker nodes in your cluster.
+Furthermore, once you want to actually use the cluster productively, we would recommend having at least 4 worker nodes in your cluster.
 "Working productively" includes working regularly with any kind of cluster, even if it's officially only a cluster for development or testing.
 You don't want your development environment to be down, because of a minor node failure.
 
-For production usage the cluster should have at least 5 nodes and a buffer of 2 nodes.
+For production usage, the cluster should have at least 5 nodes and a buffer of 2 nodes. For further information on how to size your cluster please [read this learnk8s article](https://learnk8s.io/kubernetes-node-size) which dives into all variables to consider when you don`t know the type and number of nodes to configure for your cluster(s).
```
Comment (AndiDog) [inline src/content/getting-started/sizing-multi-tenancy/index.md:42]:
```suggestion
For production usage, the cluster should have at least 5 nodes and a buffer of 2 nodes. For a deep dive on how to size your cluster, please [read this learnk8s.io article](https://learnk8s.io/kubernetes-node-size) which dives into all variables to consider when you don't know the type and number of nodes to configure for your cluster(s).
```

File: src/content/getting-started/sizing-multi-tenancy/index.md [hunk 3]
```diff
@@ -94,12 +92,12 @@ Typically, these would be 1 cluster per stage. Example:
...
+However, with a lot of teams, it might lead to a lot of wasted resources,
  especially if the products of these teams should be working together anyway.
-In these cases it might be better to have a single (or few) production (and a staging) cluster for everyone.
-The classification of these clusters should be rather by workloads then by teams.
-Thus, you could for example have a separate cluster for services that handle
- sensitive data, and/or one that holds services that need to fulfill certain compliance requirements.
+In these cases, it might be better to have a single (or few) production (and a staging) cluster for everyone.
+The classification of these clusters should be rather by workloads than by teams.
+Thus, you could, for example, have a separate cluster for services that handle
+ sensitive data, and/or one that holds services that need to fulfil certain compliance requirements.
```
Comment (AndiDog) [inline src/content/getting-started/sizing-multi-tenancy/index.md:100]:
```suggestion
 sensitive data, and/or one that holds services that need to fulfill certain compliance requirements.
```

(American English, which we mostly prefer)

**Review by AndiDog — CHANGES_REQUESTED** [review]

**Comment by pipo02mix** [comment]
@AndiDog I will wait for PR to fix linting before merging

**Review by AndiDog — APPROVED** [review]

**Pull request merged**